pkg/retriever/combinators/retrieverwithcandidatefinder.go

```diff
@@ -10,15 +10,44 @@ import (
 type RetrieverWithCandidateFinder struct {
 	CandidateFinder    types.CandidateFinder
 	CandidateRetriever types.CandidateRetriever
+	CoordinationKind   types.CoordinationKind
```

Comment:
did you work out why this was in here? I don't see the need for it here

Reply:
right. delete.